.rspec

```diff
@@ -1,2 +1,3 @@
 --color
+--backtrace
```

Comment:
✂️ 
Previous discussion here https://github.com/ministryofjustice/prison-visits-2/pull/861#discussion_r119630544

I think another option is to added to `git exclude`, so that you can override the generic project `.rspec`, although I think your preference for any project is to have this enabled so using the global rspec setting might be the best for you.